lib/kernel_tests.h

```diff
@@ -113,7 +113,7 @@ std::vector<volk_test_case_t> init_test_list(volk_test_params_t test_params)
     QA(VOLK_INIT_TEST(volk_32f_s32f_power_32f,                        test_params))
     QA(VOLK_INIT_TEST(volk_32f_sqrt_32f,                              test_params_inacc))
     QA(VOLK_INIT_TEST(volk_32f_s32f_stddev_32f,                       test_params_inacc))
-    QA(VOLK_INIT_TEST(volk_32f_stddev_and_mean_32f_x2,                test_params_inacc))
+    QA(VOLK_INIT_TEST(volk_32f_stddev_and_mean_32f_x2,                test_params.make_tol(1e-4)))
```

Comment:
So the kernels are now more accurate? That's really nice.

Reply:
Yes, they are more accurate and most of all for signals with high mean to variance ratio. That's what made me realize there was a problem in the first place. I made measurements what were too good, e.g. low variance.

Reply:
That's good.